Assignment 3.Rmd

```diff
@@ -127,19 +130,38 @@ Now we can plot our line plot using the ggplot command, "ggplot()".
 - Finally we are going to clean up our axes labels: xlab("Week") & ylab("Average Motivation")
 
 ```{r}
-
+ggplot(K6,aes(week,Average, colour=K6$cluster, xlab="Week", ylab="Average Motivation"))+geom_line()
 
 ```
 
 What patterns do you see in the plot?
+After week 3, the #2 cluster drops significantly in average motivation. However, in the same time span the #1 cluster increases. Actually, the average motivation of each cluster moves oppositely between each week. Whatever is motivating to one cluster is not motivating to the other.
 
 It would be useful to determine how many people are in each cluster. We can do this easily with dplyr.
 
 ```{r}
-K7 <- 
+K7 <- count(K5, cluster, sort=FALSE)
 ```
 
 Look at the number of people in each cluster, now repeat this process for 3 rather than 2 clusters. Which cluster grouping do you think is more informative? Write your answer below:
+I think the 3 cluster grouping is more informative because the numbers of people in the clusters are more similar. The pattern of motivation is not as obvious but having not seen what the scatter plot looks like, a more similarly-sized clustering seems to be more informative.
```

Comment:
It probably depends on what the ultimate goal is, if we were trying to maintain motivation and splitting the class into different activities perhaps the 2 clusters would work better.